bin/virtme-ssh-proxy

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+import argparse
+import array
+import socket
+
+
+def get_hostname_cid(ssh_dst):
+    """Return the hostname and CID
+
+    >>> hostname, cid = get_hostname_cid("virtme-ng/24")
+    >>> hostname
+    'virtme-ng'
+    >>> cid
+    24
+    >>> hostname, cid = get_hostname_cid("virtme/ng/21")
+    >>> hostname
+    'virtme/ng'
+    >>> cid
+    21
+
+    """
+    splitted = ssh_dst.rsplit("/", 1)
```

Comment:
Should you not use `split()` instead of `rsplit`? Because the hostname should not contain any `/`, no?

(same in the example)


Or, because we should only be called with `virtme-ng/*`, we could check if the beginning is the expected one, then skip the 10 first chars to get the CID.

Reply:
When I've implemented this I haven't had checked the hostname restrictions. 

But I'm not sure if we should add these kind of checks here since it doesn't hurt to leave as is IMO. As it's implemented the script is more generic.